pkg/cloudevents/eventcontext.go

```diff
@@ -31,6 +31,12 @@ type EventContext interface {
 	// GetType returns the CloudEvents type from the context.
 	GetType() string
 
+	// GetSource returns the CloudEvents source from the context.
+	GetSource() string
+
+	// GetSchema returns the CloudEvents schema (if any) from the context.
+	GetSchema() string
```

Comment:
The property is called SchemaURL

Reply:
OK, do you prefer a GetSchemaURL or GetSchemaUrl?

What about the method in Event. Are you OK to leave it as Schema() or do you prefer a SchemaURL/SchemaUrl?

Reply:
Go style is `GetSchemaURL`

Reply:
because `SchemaURL` conflicts with the struct member.v

Reply:
Sounds good, will do, thanks!

Reply:
Done